<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Madde::Internal::MaemoQemuSettingsWidget</class>
 <widget class="QWidget" name="Madde::Internal::MaemoQemuSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>OpenGL Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="hardwareAccelerationButton">
        <property name="text">
         <string>&amp;Hardware acceleration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="softwareRenderingButton">
        <property name="text">
         <string>&amp;Software rendering</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="autoDetectButton">
        <property name="text">
         <string>&amp;Auto-detect</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
